updated screens and content

JV0812 opened the pull request:
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

<!-- Enter the GitHub issue number or the Jira project and number (ABC-123) -->

## Select the type of change:
<!-- What types of changes does your code introduce? Select the checkbox after creating the PR. -->

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

kimsauce reviewed (dismissed):
@JV0812 You dropped in these two images, but did not add them to the Docs.

static/img/send-data/citrix-cloud-config-main.png
static/img/send-data/zoom-sign-in.png

JV0812 commented:
- static/img/send-data/citrix-cloud-config-main.png: Just replaced (v0.1.0 to v1.0.1) the config image, no file name or path change in the docs.
- static/img/send-data/zoom-sign-in.png : Added wrongly and deleted this file.
